sdk/ml/azure-ai-ml/azure/ai/ml/entities/_datastore/azure_storage.py

```diff
@@ -67,7 +67,7 @@ def __init__(
         endpoint: str = _get_storage_endpoint_from_metadata(),
         protocol: str = HTTPS,
         properties: Optional[Dict] = None,
-        credentials: Union[AccountKeyConfiguration, SasTokenConfiguration],
+        credentials: Optional[Union[AccountKeyConfiguration, SasTokenConfiguration]] = None,
```

Comment:
Is it possible to cover the change with some tests?

Reply:
I did check the tests for other datastores as well (with optional credentials already supported) to explore this possibility but there aren't any related to optional parameters in general and it doesn't seem necessary.